network/p2p/dht_test.go

```diff
@@ -28,10 +28,10 @@ func TestFindPeerWithDHT(t *testing.T) {
 	golog.SetAllLoggers(golog.LevelFatal) // change this to Debug if libp2p logs are needed
 
 	sporkId := unittest.IdentifierFixture()
-	dhtServerNodes, _ := nodesFixture(t, ctx, sporkId, 2, withDHTNodeEnabled(true))
+	dhtServerNodes, _ := nodesFixture(t, ctx, sporkId, "dht_test", 2, withDHTOptions(AsServer(true)))
```

Comment:
We may have separate `AsServer()` and `AsClient()` methods, hence no longer need to pass a boolean parameter.

Reply:
946c4ae